lib/extractor/oci-archive/layer.ts

```diff
@@ -28,12 +29,14 @@ export async function extractArchive(
 ): Promise<{
   layers: ExtractedLayers[];
   manifest: OciArchiveManifest;
+  imageConfig: DockerArchiveImageConfig;
```

Comment:
Could we maybe refactor and rename that interface just to eg `ImageConfig` if we are now using it for the OCI archive? 🙇‍♀️